DevKit/DevKit/Classes/Authentication/FormValidator.swift

```diff
@@ -0,0 +1,88 @@
+//
+//  Validator.swift
+//  DevKit
+//
+//  Created by Stanle De La Cruz on 11/12/18.
+//  Copyright © 2018 Jonathan Samudio. All rights reserved.
+//
+
+import Foundation
+
+/// FormValidator used to validate sign up forms.
+public class FormValidator {
+    
+    /// Validates the password with the given validation types and minimum Characters.
+    /// - Parameters:
+    ///   - password: The email given.
+    ///   - passwordType: The type of password.
+    public static func isValid(_ password: String, validators: [ValidationType], minCharacters: Int) -> Bool {
```

Comment:
@jgsamudio as mentioned to @StanleyOned, i tend to prefer instance functions to static ones - as static encourages the client to enact tight coupling / adds a barrier to creating a mock

Reply:
@hkellaway Using static closely resembles something that can be found in an extension function because the class does not need to store any data. However an instance will be good for mocking while testing.